<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPrograms</class>
 <widget class="QWidget" name="WidgetPrograms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="tvPrograms">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>251</width>
     <height>561</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>681</width>
     <height>551</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>679</width>
      <height>549</height>
     </rect>
    </property>
    <widget class="QToolButton" name="tbSetProgram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select new type of the program</string>
     </property>
     <property name="text">
      <string>Set program</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/magic_wand_2.png</normaloff>:/icons/magic_wand_2.png</iconset>
     </property>
     <property name="popupMode">
      <enum>QToolButton::InstantPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tbDownload">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>321</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Get list of programs from controller</string>
     </property>
     <property name="text">
      <string>Download program list from controller</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/bullet_arrow_down.png</normaloff>:/icons/bullet_arrow_down.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="tbUpload">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>321</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Put programs confguration to the controller</string>
     </property>
     <property name="text">
      <string>Upload program list to controller</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/bullet_arrow_up.png</normaloff>:/icons/bullet_arrow_up.png</iconset>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionSetUnknownProgram">
   <property name="text">
    <string>No program</string>
   </property>
   <property name="toolTip">
    <string>No program</string>
   </property>
  </action>
  <action name="actionSetTimerControlProgram">
   <property name="text">
    <string>Timer control program</string>
   </property>
   <property name="toolTip">
    <string>Timer control program</string>
   </property>
  </action>
  <action name="actionSetRelayControlProgram">
   <property name="text">
    <string>Relay control program</string>
   </property>
   <property name="toolTip">
    <string>Relay control program</string>
   </property>
  </action>
  <action name="actionSetPidControlProgram">
   <property name="text">
    <string>PID control program</string>
   </property>
   <property name="toolTip">
    <string>PID control program</string>
   </property>
  </action>
  <action name="actionResetPrograms">
   <property name="text">
    <string>Reset programs</string>
   </property>
   <property name="toolTip">
    <string>Reset programs</string>
   </property>
  </action>
  <action name="actionSetButtonControlProgram">
   <property name="text">
    <string>Button control program</string>
   </property>
   <property name="toolTip">
    <string>Button control program</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
